### PR TITLE
add to gcloud logger

### PR DESCRIPTION
**Pull request by mayadafahim**

### CODE REVIEW

File: packages/gcloud-logger/src/Logger.ts [hunk 1]
```diff
@@ -129,9 +131,12 @@ export class Logger {
 
   private stringify(message: any) {
     const excludeFn = (key: string, value: string) => {
-      if (this.excludeKeys.indexOf(key) !== -1) {
+      if (this.excludeKeys.includes(key)) {
         return '[FILTERED]';
       }
+      if (typeof value === 'object') {
+        JSON.stringify(value, excludeFn);
```
Comment (adam187) [inline packages/gcloud-logger/src/Logger.ts:138]:
```suggestion
        return JSON.stringify(value, excludeFn);
```

?

File: packages/gcloud-logger/src/Logger.ts [hunk 2]
```diff
@@ -129,9 +131,12 @@ export class Logger {
 
   private stringify(message: any) {
     const excludeFn = (key: string, value: string) => {
-      if (this.excludeKeys.indexOf(key) !== -1) {
+      if (this.excludeKeys.includes(key)) {
```
Comment (adam187) [inline packages/gcloud-logger/src/Logger.ts:134]:
```suggestion
      if (this.excludeKeys.includes(key)) {
```
I believe `indexOf` is faster

File: packages/gcloud-logger/src/Logger.ts [hunk 3]
```diff
@@ -77,6 +77,8 @@ export class Logger {
       'token',
       'newPassword',
       'oldPassword',
+      'file',
+      'resume',
```
Comment (tcichowicz) [inline packages/gcloud-logger/src/Logger.ts:81]: Can we configure the fields on particular service instead of globally?

Comment (mayadafahim) [reply packages/gcloud-logger/src/Logger.ts:81]: we can but then i will return the task in todo 😆

File: packages/gcloud-logger/src/Logger.ts (hunk 2 above)
Comment (minaeiehsan) [reply packages/gcloud-logger/src/Logger.ts]: @adam187 benchmark tells include is faster

https://www.measurethat.net/Benchmarks/ShowResult/119361

File: packages/gcloud-logger/src/Logger.ts (hunk 3 above)
Comment (kirpichenko) [reply packages/gcloud-logger/src/Logger.ts:81]: agree with @tcichowicz . doesn't seem right to put every single field inside the library.  file and resume are too specific to our domain

File: packages/gcloud-logger/src/Logger.ts (hunk 2 above)
Comment (minaeiehsan) [reply packages/gcloud-logger/src/Logger.ts]: I think it depends on the data type of array elements, btw it's no a big deal

**Comment by castarco** [comment]
@mayadafahim is it ok to close this PR?

**Comment by castarco** [comment]
stalled

**Pull request closed without merging**